packages/web/src/components/track/GiantTrackTile.tsx

```diff
@@ -531,6 +531,11 @@ export const GiantTrackTile = ({
                     strength='weak'
                     tag='h2'
                     className={cn(fadeIn)}
+                    css={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      gap: '4px'
```

Comment:
adding these props to the parent Flex no worko?

Reply:
no bc its wrapped in the text component